frontend/src/metabase/auth/components/AuthScene.jsx

```diff
@@ -16,511 +16,11 @@ class AuthScene extends Component {
-                id="bg-points"
-                stroke="#B6CCE6"
-              />
-            </svg>
-          </div>
-        </div>
-        {/*  09-12-2016 */}
-        <div
-          id="caspian"
-          className="bg-brand block py3 absolute bottom left right"
+      <section className="z1 absolute bottom left right">
+        <img
+          id="BridgeImg"
+          src="/app/img/bridge.png"
+          srcSet="/app/img/bridge.png 1x, /app/img/bridge@2x.png 2x, /app/img/bridge@3x.png 3x"
```

Comment:
Why switch this to png?

If we do need to use raster, I think these files can be optimized a bit. I ran the big one through "optipng" and it reduced by ~50%. I can push optimized versions if you want.

Reply:
That's how our illustrator provided us with the asset. Thanks for the reminder to optimize them a bit though. If you already did that I would not be offended at all if you pushed those. 👍